recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
```

Comment:
Probably not. He doesn't want to be pinged about anything other than sympy as he no longer works at Continuum. This was a recipe he added while he was still there.